### DISCUSSION TITLE
Update of docksal/solr stock image

### DISCUSSION THREAD

**Discussion by HugoCme** [docksal/docksal#1655] [Ideas]
Hi everyone,

After the recent problems about **log4j** we wanted to update a Docksal stack with a newer version of **Solr** with the stock image but we saw that it hasn't been upgraded for 1 year. 

Have you plan to update it anytime soon? Or should we look for an alternative solution?

Thanks for the good work, we really enjoy it

Regards

**Comment by lmakarov** [comment]
While [docksal/solr](https://github.com/docksal/service-solr) allowed for a straightforward and automated "Drupal Search API Solr"  integration, it became a burden to maintain (the setup process became more and more difficult to automate).

We will likely deprecate the image in favor of the [official Solr images on Docker Hub](https://hub.docker.com/_/solr) + [manual (3rd-party) setup instructions](https://www.drupal.org/docs/develop/local-server-setup/docker-development-environments/docker-with-solr-cloud-integration).

If there are alternative Solr images that do the job better than the official ones, then that's also an option.

**Reply by loopy3025** [reply]
We've switched to the ARM-supported 'solr:8.11.2' image. That doesn't throw any errors on fin start, but the  [docksal "user-owned" index](https://github.com/docksal/service-solr/blob/develop/scripts/docksal-preinit#L5) does not get created during fin start. I was wondering if it's possible the file copy is failing due to changed folder locations or something.
